### DISCUSSION TITLE
Show "X forwards, X blocks, X replies" on alias card default view & allow sort by for each

### DISCUSSION THREAD

**Discussion by jumpingbrowntrout** [simple-login/app#544] [Feature Requests]
Presently you have to select "more" on the alias card in order to see the alias stats for "X forwards, X blocks, X replies". This is tedious. Showing this on the default non expanded card view will allow users to see stats for all alias's.

Further adding the option to sort by either " forwards,  blocks,  replies" will allow users to make informed decisions on which senders they may need to unsubscribed from etc.

There is another open request asking for a table view with sort by features #520. Both my request and this would would achieve the same.  My request may be easier to implement though #520 would be a lot cleaner with large amounts of alias's.

**Comment by ageless-mushroom** [comment]
This would be great. It's already in the default view in the iOS app.

**Reply by jumpingbrowntrout** [reply]
Neat! Just checked it out on Android too and you're right it is!

**Comment by nguyenkims** [comment]
There's now an option "Always expand alias info" that allows to automatically expand alias additional info so no need to click the button "More" :)

Do you know in what case ordering by number forwards/replies/blocks are useful? I feel adding these options can make the UI too "busy" ...

**Reply by ageless-mushroom** [reply]
On the free plan with limited addresses it could be helpful to sort to see if any aliases aren't being utilized as much as the others to free up spots for different uses.

**Reply by jumpingbrowntrout** [reply]
Brilliant, love your work @nguyenkims.

Okay so here's the use case. 
Sorting by "forwards" is actually the most important for any user wishing to archive inbox-zero and/or unsubscribing from the busiest alias/sender. 
Currently the is no way to inform the users which senders/alias's are the spammiest.
I have over 1000+ domain alias that have been given out over the many years.  If we could sort by "forwards" we could use this information to either start blocking or subscribing from the senders mails.

Presently I have to download all my mail into thunderbird, use an addon to export the mbox, then write a custom python script to gather the "sender" or "alias" for each mail then perform a count the value. This is a very manual task and a decent time to run the script across the mbox.

**Reply by nguyenkims** [reply]
I see, thanks for the explanation. Please note that the number of forwards/replies are only counted in the last 14 days as email logs older than 14 days are deleted (for privacy reasons). And so the script method is actually more accurate as it counts the total number of emails received.

**Reply by jumpingbrowntrout** [reply]
Thanks for the explanation @nguyenkims - I wasn't aware it worked this way. I assumed each alias had a global counter for forwards and replies that was invoked to perform counter=counter+1 each time a forward and reply was detected. 
But it's rather tallying a count based of the current 14 day log history.

**Reply by nguyenkims** [reply]
Yes the number of forwards/replies is calculated by counting the number of rows in the email log table.
